packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.md

```diff
@@ -8824,7 +8835,7 @@ export const sizeToPixels: {
 export const Slider: React.StatelessComponent<ISliderProps>;
 
 // @public (undocumented)
-export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implements ISlider {
+export class SliderBase extends BaseComponent_2<ISliderProps, ISliderState> implements ISlider {

```

Comment:
Do you know why this was BaseComponent_2 before and why it's being changed now?

Reply:
no clue I dont think I modified anything there

Reply:
I think this was due to the imports that you had changed. Now it's fixed.